<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JobSettingsWidgetUI</class>
 <widget class="QWidget" name="JobSettingsWidgetUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QGroupBox" name="mOptionsGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Job</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="margin">
       <number>1</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>General</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout">
               <property name="spacing">
                <number>4</number>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="textLabel1">
                 <property name="text">
                  <string>Priority:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="FieldSpinBox" name="mPrioritySpin">
                 <property name="maximum">
                  <number>100</number>
                 </property>
                 <property name="field" stdset="0">
                  <string>priority</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout">
               <item>
                <widget class="QLabel" name="mPersonalPriorityLabel">
                 <property name="text">
                  <string>Personal Priority:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="FieldSpinBox" name="mPersonalPrioritySpin">
                 <property name="field" stdset="0">
                  <string>personalpriority</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="FieldCheckBox" name="mDeleteOnCompleteCheck">
                 <property name="text">
                  <string>Delete On Completion</string>
                 </property>
                 <property name="field" stdset="0">
                  <string>deleteoncomplete</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="FieldCheckBox" name="mPrioritizeOuterTasksCheck">
                 <property name="text">
                  <string>Prioritize Outer Tasks</string>
                 </property>
                 <property name="tristate">
                  <bool>false</bool>
                 </property>
                 <property name="field" stdset="0">
                  <string>prioritizeOuterTasks</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout">
               <property name="spacing">
                <number>4</number>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QCheckBox" name="mAllHostsCheck">
                 <property name="text">
                  <string>All Hosts</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="mHostListButton">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Host List</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="mEnvironmentButton">
               <property name="text">
                <string>Environment</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>16</width>
                 <height>8</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="FieldCheckBox" name="mLoggingEnabledCheck">
               <property name="text">
                <string>Enable Logging</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Notification</string>
            </property>
            <layout class="QVBoxLayout">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="margin">
              <number>2</number>
             </property>
             <item>
              <layout class="QHBoxLayout">
               <property name="spacing">
                <number>4</number>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="textLabel2_2">
                 <property name="text">
                  <string>Email:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="mEmailErrorListButton">
                 <property name="text">
                  <string>Error</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="mEmailCompleteListButton">
                 <property name="text">
                  <string>Complete</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout">
               <property name="spacing">
                <number>4</number>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="textLabel2_2_2">
                 <property name="text">
                  <string>Jabber:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="mJabberErrorListButton">
                 <property name="text">
                  <string>Error</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="mJabberCompleteListButton">
                 <property name="text">
                  <string>Complete</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="mPacketGroup">
            <property name="title">
             <string>Packets</string>
            </property>
            <layout class="QVBoxLayout">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="margin">
              <number>2</number>
             </property>
             <item>
              <layout class="QHBoxLayout">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Packet Type:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="mPacketCombo">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <item>
                  <property name="text">
                   <string>Random</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Sequential</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Continuous</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Preassigned</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Iterative</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout">
               <property name="spacing">
                <number>4</number>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="textLabel2">
                 <property name="text">
                  <string>Size:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="FieldSpinBox" name="mPacketSizeSpin">
                 <property name="maximum">
                  <number>100000</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="mUseAutoCheck">
                 <property name="text">
                  <string>Use Auto</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="textLabel_slots">
                 <property name="text">
                  <string>Slots:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="FieldSpinBox" name="mSlotsSpin">
                 <property name="maximum">
                  <number>12</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Health Checks</string>
            </property>
            <layout class="QGridLayout">
             <property name="margin">
              <number>4</number>
             </property>
             <property name="spacing">
              <number>2</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Max Task Minutes</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Max Load Minutes</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="FieldSpinBox" name="mMaxLoadTimeSpin">
               <property name="maximum">
                <number>999999</number>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_maxquiet">
               <property name="text">
                <string>Max Quiet Minutes</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="FieldSpinBox" name="mMaxQuietTimeSpin">
               <property name="maximum">
                <number>999999</number>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="FieldSpinBox" name="mMaxTaskTimeSpin">
               <property name="maximum">
                <number>999999</number>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="FieldSpinBox" name="mMinMemorySpin">
               <property name="maximum">
                <number>999999999</number>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_minMem">
               <property name="text">
                <string>Min Memory (MB)</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="FieldSpinBox" name="mMaxMemorySpin">
               <property name="maximum">
                <number>999999999</number>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Max Memory (MB)</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="FieldSpinBox" name="mMaxErrorsSpin">
               <property name="maximum">
                <number>999999999</number>
               </property>
               <property name="field" stdset="0">
                <string>maxerrors</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_maxerrors">
               <property name="text">
                <string>Max Errors</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Services</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="ServiceCheckTree" name="mServiceTree"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>47</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>31</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="mApplyInstantSettings">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mResetInstantSettings">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="mCustomJobSettingsStack">
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mEmptyPage"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FieldSpinBox</class>
   <extends>QSpinBox</extends>
   <header>fieldspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>FieldCheckBox</class>
   <extends>QCheckBox</extends>
   <header>fieldcheckbox.h</header>
  </customwidget>
  <customwidget>
   <class>ServiceCheckTree</class>
   <extends>QTreeView</extends>
   <header>servicechecktree.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
